examples/ingest/weaviate/ingest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Uploads the structured output of the files within the given S3 path to a Weaviate index.
+
+# Structured outputs are stored in s3-small-batch-output-to-weaviate/
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+cd "$SCRIPT_DIR"/../../.. || exit 1
+
+PYTHONPATH=. ./unstructured/ingest/main.py \
+        s3 \
```

Comment:
can we use the local connector for the example to match what we do in our ingest destination tests